Makefile

```diff
@@ -1,4 +1,9 @@
+CRUXDB=$(shell docker ps -aq --filter name=CruxDB)
+
 crux:
+ifneq ($(strip $(CRUXDB)),)
```

Comment:
Actually I would only remove it if is not running. Is way easier to test `query` and `entity` when there is a long running container. Other solution would be to create a new target `reset-crux`.

Reply:
> Actually I would only remove it if is not running. Is way easier to test `query` and `entity` when there is a long running container. Other solution would be to create a new target `reset-crux`.

Done. Will not remove the Docker instance if existing `CruxDB` instance still running.